sound/soc/sof/intel/hda-loader.c

```diff
@@ -25,6 +25,44 @@
 
 #define HDA_CL_STREAM_FORMAT 0x40
 
+static void hda_set_ssps_slave(struct snd_sof_dev *sdev)
```

Comment:
maybe call this function hda_ssp_resume()? then it will avoid using "slave" and it will leave the door open to move other things into it

Reply:
Good suggestion, let me change them.